src/Forms/FormRequestHandler.php

```diff
@@ -77,7 +77,6 @@ public function __construct(Form $form)
     public function Link($action = null)
     {
         // Forms without parent controller have no link;
-        // E.g. Submission handled via graphql
```

Comment:
That's still valid - graphql is still supported and submissions handled via graphql from the front-end will presumably meet this criterion

Reply:
I don't think it was ever valid - graphql was not used to submit forms